pkg/virt-controller/watch/migration.go

```diff
@@ -727,6 +727,14 @@ func (c *MigrationController) createTargetPod(migration *virtv1.VirtualMachineIn
 		c.podExpectations.CreationObserved(key)
 		return err
 	}
+
+	// If VM was not paused patch the pod with unpaused condition
+	// to allow readiness gate to catch up before live migration is finished
+	if err := syncPausedConditionToPod(c.clientset, vmi, pod); err != nil {
+		c.recorder.Eventf(vmi, k8sv1.EventTypeWarning, FailedCreatePodReason, "Error syncing paused condition at migration at vmi target pod: %v", err)
+		return fmt.Errorf("failed to sync paused condition at vmi migration target pod: %v", err)
+	}
+
```

Comment:
this needs to be moved outside of the createTargetPod function. The reason is, the create could be successful, but this patch could fail. The patch will never be eventually consistent in that case because this function isn't called if the pod already exists.

Reply:
I will move it to `updateStatus`